src/Compilers/CSharp/Test/Symbol/Symbols/Source/ExpressionBodiedPropertyTests.cs

```diff
@@ -504,5 +504,27 @@ class C
             Assert.True(p.IsExpressionBodied);
             Assert.Equal(RefKind.Ref, p.GetMethod.RefKind);
         }
+
+        [Fact]
+        public void ReadonlyRefReturningExpressionBodiedProperty()
+        {
+            var comp = CreateCompilationWithMscorlib45(@"
+class C
+{
+    int field = 0;
+    public readonly ref int P => ref field;
+}");
+            comp.VerifyDiagnostics();
+
+            var global = comp.GlobalNamespace;
+            var c = global.GetTypeMember("C");
+
+            var p = c.GetMember<SourcePropertySymbol>("P");
+            Assert.Null(p.SetMethod);
+            Assert.NotNull(p.GetMethod);
+            Assert.False(p.GetMethod.IsImplicitlyDeclared);
+            Assert.True(p.IsExpressionBodied);
+            Assert.Equal(RefKind.Ref, p.GetMethod.RefKind);
```

Comment:
This is correct for now. Binding is NYI and falls back on regular ref returns.
The test will conveniently fail when binding is implemented.

---
In reply to: [102295224](https://github.com/dotnet/roslyn/pull/17223#discussion_r102295224) [](ancestors = 102295224)